tests/unit/gensym_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest 
+
+from lvm_gensym import * 
+
+used_vg_names = ['rhel_tflannag_bos_csb', 'rhel_tflannag_bos_csb_0', 'rhel_tflannag_bos_csb_1']
```

Comment:
Make to remove any of these that look like hostnames. I'd remove your username as well.